bats/fb-test-foreman.bats

```diff
@@ -16,3 +18,11 @@ set -o pipefail
 @test "check smart proxy is registered" {
   hammer proxy info --name=$(hostname -f)
 }
+
+@test "Assign proxy to default taxonomies" {
```

Comment:
This looks like it doesn't matter anymore and we should actually drop this workaround on 1.23+